.github/workflows/macos_tests.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
           mkdir -p build/cmake
           cmake -S $(pwd) -B build/cmake -GNinja -DCMAKE_BUILD_TYPE=RelWithDebInfo \
             -DCMAKE_PREFIX_PATH=${{ github.workspace }}/qt_dist/lib/cmake
-          cmake --build build/cmake -j$(nproc) --target dummyvpn
```

Comment:
This is a sneaky one that keeps catching us out every now and then... `nproc` doesn't exist on MacOS, so this kind of expression expands to `cmake --build build/cmake -j --target dummyvpn` which gets interpreted as either zero or infinite parallelism - neither of which produces speedy builds.